packages/xstate-react/src/index.ts

```diff
@@ -1,4 +1,4 @@
-export { useMachine, asEffect, asLayoutEffect } from './useMachine';
```

Comment:
Those became hard to support after the changes. Maybe I could make them work after digging into this more but since we've wanted to remove those anyway... I've decided to just do it here.

It means that after landing this we should update the `@xstate/react`'s version

Reply:
Are we still going to export `useMachine`?

Reply:
Yeah, the export is still here :)